utils/submit_tracks.py

```diff
@@ -24,29 +32,48 @@ class TrackData(TypedDict):
     trigger: list[str]
     type: str
 
-
-class CSVRow(TypedDict):
-    desc: str
-    name: str
-    sources: list[str]
-    urls: list[str]
-
-
-CSVData = dict[str, CSVRow]
-CSVCollection = dict[str, CSVData]
+# Datamodel for track descriptions
+class DescData(TypedDict):
+    description: str
+    source_names: list[str]
+    source_urls: list[str]
+    track_name: NotRequired[str]
+
+Descriptions = dict[str, DescData]
+DescCollection = dict[str, Descriptions]
+
+# Global variables / constants
+suf = "/nfs/public/ro/enswbsites_codon/newsite"
```

Comment:
Could you use more clear variable names? what is suf and pref?

Reply:
suffix and prefix

Reply:
Of what?

Reply:
These are used as suffix/prefix in datadir paths (see the lines below) to avoid repetition.

Reply:
path_prefix
path_ppostfix